pymc3/distributions/continuous.py

```diff
@@ -2651,6 +2643,17 @@ def __init__(self, nu, *args, **kwargs):
         super().__init__(alpha=nu / 2.0, beta=0.5, *args, **kwargs)
 
 
+class WeibullBetaRV(WeibullRV):
+    ndims_params = [0, 0]
+
+    @classmethod
+    def rng_fn(cls, rng, alpha, beta, size):
+        return beta * rng.weibull(alpha, size=size)
```

Comment:
Instead of creating a class that's just a constant times an existing `RandomVariable`, let's just create the product in `Distribution.dist`.

Reply:
How will we pass the beta argument to the logp/logcdf?

Reply:
Since this is just a scaled random variable, we would need to write the `logpt` dispatch that formulates the log-likelihood for a scale random variable.

Reply:
That goes towards the idea of having that automatic logp inference for continuous transformations.

I think it's a great functionality to add but I think it's deserves it's own PR. Some work has to be done to figure out how it would fit with the automatic variable registration and automatic transformations inside the model object.

I am happy to either remove this distribution from the PR or add a clear TODO comment above it for now.

Reply:
I'm guessing it can be done in a single commit, so it might as well go here&mdash;or any other PR that needs it.

Reply:
Will take care here: https://github.com/pymc-devs/pymc3/pull/4653